CHANGELOG.md

```diff
@@ -31,7 +74,7 @@ Thanks [@SUPERCILEX](https://github.com/SUPERCILEX) for adding great new feature
 
 ### Enhancement
 
-- Add `Licenses#fromGitHubApacheV2(String)`, `Licenses#fromGitHubApacheV2(String, String)`, `Licenses#fromGitHubMIT(String)`, `Licenses#fromGitHubMIT(String, String)`, `Licenses#fromGitHubBSD(String)`, `Licenses#fromGitHubBSD(String, String)`.  
```

Comment:
this 2 white spaces at the end of line should be preserved(for new line)

Reply:
Oh, that's a cool trick! IntelliJ keeps trying to autoformat those spaces away so feel free to ping me if this change gets added again. 😄